ee/maintained-apps/ingesters/homebrew/scripts.go

```diff
@@ -326,7 +328,9 @@ sudo installer -pkg "$TMPDIR"/%s -target / -applyChoiceChangesXML "$CHOICE_XML"
 // Symlink writes a command to create a symbolic link from 'source' to 'target'.
 func (s *scriptBuilder) Symlink(source, target string) {
 	pathname := filepath.Dir(target)
-	s.Writef(`[ -d "%s" ] && /bin/ln -h -f -s -- "%s" "%s"`, pathname, source, target)
+	s.Writef(`if [ -d "%s" ]; then
```

Comment:
Do we want to let these pass if the dir doesn't exist? Thinking we _don't_, unlike the backup case where if Docker isn't installed to begin with we don't need to move a nonexistent dir around.